Int overflow on 32 bits arches

Golang 1.12.6 on i686 and armv7:

```
Testing    in: /builddir/build/BUILD/gocql-b99afaf3b1639e5264f5e1cabf04fdb2e5ed57ac/_build/src
         PATH: /builddir/build/BUILD/gocql-b99afaf3b1639e5264f5e1cabf04fdb2e5ed57ac/_build/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/sbin
       GOPATH: /builddir/build/BUILD/gocql-b99afaf3b1639e5264f5e1cabf04fdb2e5ed57ac/_build:/usr/share/gocode
  GO111MODULE: off
      command: go test -buildmode pie -compiler gc -ldflags "-X github.com/gocql/gocql/version.commit=b99afaf3b1639e5264f5e1cabf04fdb2e5ed57ac -X github.com/gocql/gocql/version=0 -extldflags '-Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld '"
      testing: github.com/gocql/gocql
github.com/gocql/gocql
PASS
ok  	github.com/gocql/gocql	3.874s
github.com/gocql/gocql/internal/lru
PASS
ok  	github.com/gocql/gocql/internal/lru	0.015s
github.com/gocql/gocql/internal/murmur
FAIL	github.com/gocql/gocql/internal/murmur [build failed]
BUILDSTDERR: # github.com/gocql/gocql/internal/murmur [github.com/gocql/gocql/internal/murmur.test]
BUILDSTDERR: ./murmur_test.go:131:36: constant 7627370222079200297 overflows int
```

--- a/internal/murmur/murmur_test.go
+++ b/internal/murmur/murmur_test.go
@@ -127,8 +127,8 @@ func BenchmarkMurmur3H1(b *testing.B) {
 	b.RunParallel(func(pb *testing.PB) {
 		for pb.Next() {
 			h1 := Murmur3H1(data)
-			if h1 != 7627370222079200297 {
-				b.Fatalf("expected %d got %d", 7627370222079200297, h1)
+			if h1 != int64(7627370222079200297) {
+				b.Fatalf("expected %d got %d", int64(7627370222079200297), h1)
 			}
 		}
 	})

